transmission-nat-pmp/root/etc/s6-overlay/s6-rc.d/svc-mod-transmission-nat-pmp/run

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/with-contenv bash
 
-GATEWAY=$(grep nameserver /etc/resolv.conf | awk '{print $2}')
+# get nat-pmp gateway IP address using the NAT_PMP_GATEWAY env variable or 
+# use the system nameserver as fallback
+if [ -z "${NAT_PMP_GATEWAY}" ]; then
+	GATEWAY=$(grep nameserver /etc/resolv.conf | awk '{print $2}')
+else 
+	GATEWAY=${NAT_PMP_GATEWAY}
+fi
```

Comment:
Just to make things simpler, thoughts on allowing the user to set the `GATEWAY` environment variable instead? If it's not provided by the user, then we use nameserver:

For example:

```
if [[ -z "${GATEWAY}" ]]; then
	GATEWAY=$(grep nameserver /etc/resolv.conf | awk '{print $2}')
fi
```

Reply:
Sure thing! Made the requested change and tested it, working well for me.